core/fetcher/graffiti_internal_test.go

```diff
@@ -0,0 +1,167 @@
+// Copyright © 2022-2025 Obol Labs Inc. Licensed under the terms of a Business Source License 1.1
+
+package fetcher
+
+import (
+	"crypto/rand"
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/obolnetwork/charon/app/version"
+	"github.com/obolnetwork/charon/core"
+	"github.com/obolnetwork/charon/testutil"
+)
+
+// randomString returns a random string of length n.
+func randomString(n int) string {
```

Comment:
We have other random test functions in `random.go` like `RandomBytes32`. We can move this function there so others can reuse it.

Reply:
And the name shall say Bytes as it generates bytes, likely non-printable.